README.md

```diff
@@ -43,24 +43,33 @@ forge script ./script/DeployCOWFeeModule.s.sol \
 ```
 
 ### Verify module
+
 To verify the module if the contract is already deployed:
+
 ```sh
 source .env
 
 forge verify-contract <fee-module-address> COWFeeModule --chain-id <chain-id> --etherscan-api-key $ETHERSCAN_API_KEY --constructor-args $(cast abi-encode "constructor(address,address,address,address,bytes32,address,uint256)" $SETTLEMENT $TARGET_SAFE $WRAPPED_NATIVE_TOKEN $KEEPER $APP_DATA $RECEIVER $MIN_OUT)
 ```
 
+You can compare the parameters of a new module with those of the module that is currently registered in the CoW multisig responsible for fee withdrawals.
```

Comment:
nit: is not this a new section? 

Mayne "verify module" can be renamed to "verify contract in block explorers" or similar, and this section is "Verify config params for a new module"